packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.js

```diff
@@ -6,11 +6,10 @@ const isDefined = item => item !== undefined;
 const getAxesDomains = axes =>
```

Comment:
Rename this function OR remove it. Now it does nothing

Reply:
What about 'collectAxesTypes' name?

Reply:
nice 👍